.github/actions/artifact_failure/action.yml

```diff
@@ -7,15 +7,19 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: pack failure artifacts
+      shell: bash
+      run: |
+        tar --exclude='target' \
+            --exclude='website' \
+            --exclude='docs' \
+            --exclude='docker' \
+            --exclude='bins' \
+            --exclude='.git' \
+            --exclude='tests/suites/0_stateless/13_tpch/data' \
+            --exclude='failure-*' \
+            -zcf failure-${{ inputs.name }}.tar.gz .
     - uses: actions/upload-artifact@v3
       with:
         name: ${{ inputs.name }}
-        path: |
-          .
-          !./target
-          !./website
-          !./docs
-          !./docker
-          !./bins
-          !./.git
-          !./tests/suites/0_stateless/13_tpch/data
+        path: failure-${{ inputs.name }}.tar.gz
```

Comment:
BTW, I prefer to use `zstd` for better compression rate (

Reply:
We do not have a zstd installed on runner currently, maybe we can change it later.
But the artifacts are actually small enough to ignore the compression rate, and `tar.gz` is more convenient for developers.